src/components/Common/PreventRefresh.js

```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react'
+
+export class PreventRefresh extends Component {
+  onUnload = (e) => {
+    e.returnValue = "You'll loose current deployment, are you sure?"
```

Comment:
This message is not the one that is being shown.

Besides, I think it should be `You'll lose your current deployment, are you sure?`

Reply:
Typo fixed.

For the alert to be displayed it's required to set some value for `returnValue`, despite it's not being used by browsers: 
1. https://www.chromestatus.com/feature/5349061406228480 
2. https://developer.mozilla.org/en-US/docs/Web/Events/beforeunload